store/etcd/rulesets.go

```diff
@@ -29,6 +30,7 @@ type RulesetService struct {
 }
 
 // List returns all the rulesets entries under the given prefix.
+// If the prefix is empty, it returns **all** the rulesets entries.
```

Comment:
That's not true, it always returns a subset of the rulesets because of the limit

Reply:
I've updated the comment to be more specific in https://github.com/heetch/regula/pull/43/commits/2c44b1d561f58e7a90093e8208ac35569dc337b0. Tell me if it suits you

Reply:
💥

Reply:
Good catch btw @asdine

Reply:
Fixed here https://github.com/heetch/regula/pull/43/commits/0087813e243ac36457c9ddcd9acf32753a6b5787